### PR TITLE
DOC-5495: Updated PostgreSQL section in CENM databases doc

### PR DESCRIPTION
**Pull request by amiegrace12**
- Added options for creating a CENM service instance user in PostgreSQL using a custom schema name.
- Added escape characters to *\<host\>*,*\<port\>* and *\<sid\>* as they weren't showing on the docs site.

Preview built here: https://amie.preview.docs.r3.com/en/platform/corda/1.5/cenm/database-set-up.html#postgresql

### CODE REVIEW

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/1.5/cenm/database-set-up.md [hunk 1]
```diff
@@ -546,7 +546,7 @@ database = {
 }
 ```
 
-Replace the placeholders *\<host\>*,*\<port\>* and *\<sid\>* with appropriate values. For a basic Oracle installation, the default *\<sid\>* value is `xe`.
+Replace the placeholders *host*,*port* and *sid* with appropriate values. For a basic Oracle installation, the default *sid* value is `xe`.
```
Comment (yarivus) [inline content/en/platform/corda/1.5/cenm/database-set-up.md:549]:
```suggestion
Replace the placeholders *host*, *port* and *sid* with appropriate values. For a basic Oracle installation, the default *sid* value is `xe`.
```

**Review by yarivus — CHANGES_REQUESTED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

**Review by jonathan-jacques — APPROVED** [review]
I approve the changes.

File: content/en/platform/corda/1.5/cenm/database-set-up.md [hunk 2]
```diff
@@ -261,8 +261,19 @@ GRANT USAGE, SELECT ON ALL sequences IN SCHEMA "my_schema" TO "my_user";
 ALTER DEFAULT privileges IN SCHEMA "my_schema" GRANT USAGE, SELECT ON sequences TO "my_user";
 ALTER ROLE "my_user" SET search_path = "my_schema";
 ```
+If you are creating a CENM service instance user in PostgreSQL using a custom schema name (different from the username), connect to the database as an administrator and run the script above. The last statement in the script - setting the `search_path` - prevents querying the differing [default schema search path](https://www.postgresql.org/docs/9.3/static/ddl-schemas.html#DDL-SCHEMAS-PATH). If you don't have the privileges required to run the script, set the custom schema in the URL in the database configuration section of the node.conf file:
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.5/cenm/database-set-up.md:264]:
```suggestion
If you are creating a CENM service instance user in PostgreSQL using a custom schema name (different from the user name), connect to the database as an administrator and run the script above. The last statement in the script - setting the `search_path` - prevents querying the differing [default schema search path](https://www.postgresql.org/docs/9.3/static/ddl-schemas.html#DDL-SCHEMAS-PATH). If you don't have the privileges required to run the script, set the custom schema in the URL in the database configuration section of the `node.conf` file: 
```

File: content/en/platform/corda/1.5/cenm/database-set-up.md [hunk 3]
```diff
@@ -261,8 +261,19 @@ GRANT USAGE, SELECT ON ALL sequences IN SCHEMA "my_schema" TO "my_user";
...
+database = {
+    jdbcDriver = path/to/postgresql-xx.x.x.jar
+    driverClassName = "org.postgresql.Driver"
+    url = "jdbc:postgresql://<host>:<port>/sample_db?currentSchema=my_schema"
+    user = my_user
+    password = "my_password"
+}
+```
+
+Replace the placeholders *\<host\>* and *\<port\>* in the URL with appropriate values.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.5/cenm/database-set-up.md:276]:
```suggestion
Replace the placeholders `\<host\>` and `\<port\>` in the URL with appropriate values.
```

File: content/en/platform/corda/1.5/cenm/database-set-up.md [hunk 4]
```diff
@@ -504,7 +515,7 @@ database = {
 }
 ```
 
-Replace the placeholders *<host>* and *<port>* with appropriate values (the default SQL Server port is 1433). By default, the connection to the database is not SSL. To secure the JDBC connection, refer to [Securing JDBC Driver Applications](https://docs.microsoft.com/en-us/sql/connect/jdbc/securing-jdbc-driver-applications?view=sql-server-2017).
+Replace the placeholders *\<host\>* and *\<port\>* with appropriate values (the default SQL Server port is 1433). By default, the connection to the database is not SSL. To secure the JDBC connection, refer to [Securing JDBC Driver Applications](https://docs.microsoft.com/en-us/sql/connect/jdbc/securing-jdbc-driver-applications?view=sql-server-2017).
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.5/cenm/database-set-up.md:518]:
```suggestion
Replace the placeholders `\<host\>` and `\<port\>` with appropriate values (the default SQL Server port is 1433). By default, the connection to the database is not SSL. To secure the JDBC connection, refer to [Securing JDBC Driver Applications](https://docs.microsoft.com/en-us/sql/connect/jdbc/securing-jdbc-driver-applications?view=sql-server-2017).
```

File: content/en/platform/corda/1.5/cenm/database-set-up.md [hunk 5]
```diff
@@ -545,7 +556,7 @@ database = {
 }
 ```
 
-Replace the placeholders *<host>*, *<port>*, and *<sid>* with appropriate values. For a basic Oracle installation, the default *<sid>* value is `xe`.
+Replace the placeholders *\<host\>*, *\<port\>*, and *\<sid\>* with appropriate values. For a basic Oracle installation, the default *\<sid\>* value is `xe`.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.5/cenm/database-set-up.md:559]:
```suggestion
Replace the placeholders `\<host\>`, `\<port\>`, and `\<sid\>` with appropriate values. For a basic Oracle installation, the default `\<sid\>` value is `xe`.
```

File: content/en/platform/corda/1.5/cenm/database-set-up.md [hunk 6]
```diff
@@ -562,14 +573,13 @@ See below an example CENM service configuration for PostgreSQL:
...
-    url = "jdbc:postgresql://<host>:<port>/<database>"
+    url = "jdbc:postgresql://<host>:<port>/sample_db?currentSchema=my_schema"
     user = my_user
     password = "my_password"
-    schema = my_schema
 }
 ```
 
-Replace the placeholders *<host>*, *<port>*, and *<database>* with appropriate values.
+Replace the placeholders *\<host\>* and *\<port\>* with appropriate values.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.5/cenm/database-set-up.md:582]:
```suggestion
Replace the placeholders `\<host\>` and `\<port\>` with appropriate values.
```

**Review by AnnaK-R3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @amiegrace12

**Pull request merged**